app/assets/javascripts/leaving_dad.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
+# All this logic will automatically be available in application.js.
+# You can use CoffeeScript in this file: http://coffeescript.org/
```

Comment:
can we no longer use .coffee
it was a mistake
can this just be js

Reply:
i removed this